src/CoreBundle/Resources/public/javascript/cms.js

```diff
@@ -527,7 +527,19 @@ CHAMELEON.CORE.initializeEntryAutocomplete = function($element) {
         },
         afterSelect: function(item) {
             if (typeof item.id !== "undefined") {
-                if ("" !== $element.data('record-url')) {
+                var handled = false;
+                if ("" !== item.id) {
+                    // Look for a table row for this entry and its click handler:
+
+                    var $row = $("tr[data-record-id='"+item.id+"']");
+                    var $onClicks = $row.find("td[onClick]");
+                    if ($row.length > 0 && $onClicks.length > 0) {
+                        $($onClicks[0]).click();
+                        handled = true;
+                    }
+                }
+                if (false === handled && "" !== $element.data('record-url')) {
+                    // TODO could take the href of the above row td or simply click there?
```

Comment:
I would use the url so other events are not triggered here. Or should they? I'm not sure. We don't have other events on cells at the moment, but what about inline editing later maybe?

Reply:
Or on-clicks being not defined as "onclick" (but with a proper listener)?

Reply:
Just removed the TODO for now:

The existing "record-url" function of an autocomplete field is preserved.

All other functions can be added/changed when needed.